HandleNHNotifyMessageCallOrchestrator/handler.ts

```diff
@@ -29,7 +32,10 @@ export type NhNotifyMessageOrchestratorCallInput = t.TypeOf<
 >;
 
 interface IHandlerParams {
-  notifyMessageActivity: CallableActivity<NotifyMessageActivityBodyImpl>;
+  notifyMessageActivity: CallableActivity<
+    NotifyMessageActivityInput,
+    NotifyMessageActivityResultSuccess
```

Comment:
Why the need of specifying also the success type, since it's a simple `ActivityResultSuccess`?

Reply:
Just for more consistency. As we lost the `ActivityBody` here, there's no other mechanism that is ensuring us to use the very same output type of the activity. 

Thinking about making `S` non-optional for `callableActivity`....

Reply:
We could think about exporting the callableActivity directly from  the activity itself, what do you think? 
So it's up to the activity expose it with right types

Reply:
Or more: we could just scaffold all types so that we don't have to use so much generics